core/domain/role_services_test.py

```diff
@@ -0,0 +1,116 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test functions relating to roles and actions."""
+
+from core.domain import role_services
+from core.tests import test_utils
+import feconf
+
+
+class RoleDomainUnitTests(test_utils.GenericTestBase):
+    """Tests for PARENT_ROLES and ROLE_ACTIONS."""
+    PARENT_ROLES = role_services.PARENT_ROLES
+    ACTIONS = role_services.ROLE_ACTIONS
+
+    def test_dicts_have_same_keys(self):
+        """Test that PARENT_ROLES and ROLE_ACTIONS have same keys.
+        """
```

Comment:
This can go at the end of the previous line, right? (For one-line docstrings we generally do that.)